mlir/tools/mlir-miopen-lib/mlir-miopen-lib.cpp

```diff
@@ -124,51 +129,102 @@ extern "C" MlirHandle CreateMlirHandle(const char *arguments) {
         strToInt("padding_h"), strToInt("padding_w"), module, builder,
         argMap["kernel_name"], mlir::FloatType::getF32(&(handle->context)),
```

Comment:
Guess you will need to consume the type in module generation.

Reply:
I will go ahead and add that.